### PR TITLE
Add Chikoo.app Latest

### PR DESCRIPTION
**Pull request by rqelibari**
After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

Add a cask for Chikoo app, a document/library management app.

### CODE REVIEW

File: Casks/chikoo.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'chikoo' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://codingturtle.com/chikoo/download'
```
Comment (vitorgalvao) [inline Casks/chikoo.rb]: `url "https://download.codingturtle.com/download-files/chikoo/chikoo-#{version}.zip"`.

File: Casks/chikoo.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'chikoo' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/chikoo.rb]: `1.1`.

File: Casks/chikoo.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/chikoo.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/chikoo.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/chikoo.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) with `https://download.codingturtle.com/appcast/chikoo/11000`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/chikoo.rb (hunk 1 above)
Comment (rqelibari) [inline Casks/chikoo.rb]: I will add all those things. But how did you find out, that there was an `appcast` url?
And there was a versioned download link? On the website, there is no information about all that.

Comment (rqelibari) [inline Casks/chikoo.rb]: The only page related to releases I found is: [Release Notes](https://codingturtle.com/chikoo/releasenotes)

Comment (vitorgalvao) [inline Casks/chikoo.rb]: > And there was a versioned download link?

The previous link you had was a simple redirect. Open it in Chrome, for example, and it’ll show you the true link in the Downloads page.

> But how did you find out, that there was an `appcast` url?

We [have a script](https://github.com/caskroom/homebrew-cask/blob/master/developer/bin/find_sparkle_appcast).

**Pull request merged**